test.py

```diff
@@ -563,16 +563,65 @@ def run_test(framework, cluster, repo_root):
         logger.info(msg, runtests_script, framework.name)
         raise CommandFailure(cmd_args)
 
+def setup_junit_xml():
+    """Try to download and import junit_xml"""
+    workdir = get_work_dir()
+    lib_dir = os.path.join(workdir, 'lib', 'python')
+    pip_cmd = ['pip', 'install', 'junit-xml', '-t', lib_dir]
```

Comment:
On looking to implement this; it's messy.  venvutil knows about virtualenvs which I've always found overcomplicated and now discovering are more fragile than I realized; so one of my goals is to only have one for most runs, and none for this failsafe purpose.

Having a module that knows about virtualenv specific things for downloading seems weird; venvutil already covers that.  So I just put in a lame convenience method to module_downloader that calls through.